docpages/building/openbsd.md

```diff
@@ -0,0 +1,126 @@
+\page buildopenbsd Building on OpenBSD
+
+## 1. Toolchain
+Since the project uses `CMake` and `git`, you'll need to install them! If you don't have them, you can do the following:
+
+```bash
+pkg_add cmake git
+```
+
+Once that's done, check the version of CMake!
+
+```bash
+cmake --version
+cmake version 3.25.2
+```
+
+If your CMake version is not as shown above then don't worry! You can still follow along, even if you're ahead or behind!
+
+## 2. Install Voice Dependencies (Optional)
+If you wish to use voice support, you'll need to do the following:
+
+```bash
+pkg_add libsodium opus pkgconf
+```
+
+## 3. Create a CMake project
+
+In an empty directory, create the following files and directories:
+
+\dot
+digraph "Example Directory" {
+	graph [ranksep=1];
+	node [colorscheme="blues9", fontname="helvetica"];
+	
+	"Your Directory" [style=filled, color=1, shape=rect]
+    
+    	subgraph cluster_0 {
+		style=filled;
+		color=lightgrey;
+		node [style=filled, color=2, shape=rect]
+		build;
+		libs;
+		src;
+		"CMakeLists.txt";
+		label = "The main area for your bot's files.";
+	}
+	
+	subgraph cluster_1 {
```

Comment:
these pages are for how to build dpp not a bot

Reply:
Ah right yeah i'll move it :)